src/resources/postcss/views/month/_multiday-event.pcss

```diff
@@ -5,10 +5,26 @@
 	 * ----------------------------------------------------------------------------- */
 
 	.tribe-events-calendar-month__event-multiday {
-		border-radius: 6px;
-		margin-bottom: var(--spacer-0);
-		background-color: white;
-		overflow: hidden;
+		display: none;
+		min-width: calc(100% + 1px);
```

Comment:
let's set this as `width`, as `min-width` will have unintended `width` consequences if we have long content.

Reply:
The thing when I set that to width is that it was defaulting the rest (even with modifiers) to 100%

Reply:
it sounds like you've got competing styles somewhere. given the order of this CSS document, the `--width-#` modifiers come after this class, so it should override this style if using `width: calc(100% + 1px);`.

Reply:
Perfect, looking at that 👍